cmd/fs-v1-multipart.go

```diff
@@ -79,11 +79,17 @@ func (fs *FSObjects) backgroundAppend(ctx context.Context, bucket, object, uploa
 	file.Lock()
 	defer file.Unlock()
 
+	// Preserving the decompressed part size in the append file.
+	if partId > 0 {
+		file.compressParts = append(file.compressParts, CompressPartInfo{PartNumber: partId,ActualSize: actualSize})
+	}
+
 	// Since we append sequentially nextPartNumber will always be len(file.parts)+1
 	nextPartNumber := len(file.parts) + 1
 	uploadIDDir := fs.getUploadIDDir(bucket, object, uploadID)
 
 	entries, err := readDir(uploadIDDir)
+	
```

Comment:
do not add unexpected space changes, as it adds into the PR and hinders the review.